.github/workflows/windows.yaml

```diff
@@ -0,0 +1,43 @@
+# This workflow is for builds on Windows. We cannot use the existing
+# Travis-CI workflow as that depends on already having functioning
+# BuildKit to run the v1.1-experimental Dockerfile that builds BuildKit.
+name: Windows
+
+on: [push, pull_request]
```

Comment:
Maybe something like https://github.com/tonistiigi/fsutil/pull/88/files#diff-e9f950f17198d3d5e3122a44230a09b9R3 is more appropriate

Reply:
I had something like that earlier (also matching v* and docker-* for the release branches), but it seemed a bit annoying to have to either temporarily change it when I am working on something, or create a PR, just to see the builds. I noticed that the Travis config doesn't restrict itself either, which has been very useful in the past.

Plus if I have to create PRs to see CI results, that gets accounted against moby/. I believe a push gets accounted against my own branch, for whatever rate-limiting GitHub applies to free projects.

Reply:
I left `gh_test_ci` name there to test without pushing.

Actually, not a big issue. I remember in `tonistiigi/fsutil` one problem I had was that I don't have a fork and need to push to the origin for PRs. That does not exist in here.